src/test/java/roomescape/repository/reservation/ReservationDatabaseRepositoryTest.java

```diff
@@ -14,18 +16,30 @@
 import org.springframework.test.context.ActiveProfiles;
 import roomescape.domain.reservation.Reservation;
 import roomescape.domain.reservation.ReservedDateTime;
+import roomescape.domain.time.Time;
+import roomescape.repository.TimeDatabaseRepository;
 
 @JdbcTest
-@Import(ReservationDatabaseRepository.class)
+@Import({ReservationDatabaseRepository.class, TimeDatabaseRepository.class})
 @ActiveProfiles(value = "test")
 class ReservationDatabaseRepositoryTest {
 
     @Autowired
     private ReservationDatabaseRepository reservationDatabaseRepository;
 
+    @Autowired
+    private TimeDatabaseRepository timeDatabaseRepository;
+
     private static final LocalDate reservedDate = LocalDate.now().plusDays(10);
```

Comment:
[제안 🗣️ ]

상수는 컨벤션으로 대문자를 씁니다!
```suggestion
    private static final LocalDate RESERVED_DATE = LocalDate.now().plusDays(10);
```